src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -31,7 +31,7 @@ public interface IClusterMessage
     /// <summary>
     /// Cluster commands sent by the USER via <see cref="Cluster"/> extension.
     /// </summary>
-    internal class ClusterUserAction
+    internal static class ClusterUserAction
```

Comment:
This class is never instantiated, should be made `static` to resolve some compiler grumbling.